.setup/bin/setup_sample_courses.py

```diff
@@ -508,8 +505,6 @@ def create_gradeable_submission(src, dst):
     a .zip), then we will extract the contents of the source (using ZipFile) to the destination, else we
```

Comment:
I was actually talking about fixing this line as it says "using ZipFile" which after your pr will no longer be used. When you removed the 3 lines at the top it switched your line count so when I said line 508 I was talking about before you removed the lines not after.

```suggestion
    a .zip), then we will extract the contents of the source (using shutil) to the destination, else we
```

Reply:
Did that. Thanks for pointing out